packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -501,6 +502,7 @@ export const linkedAccountRoutes = (
               .string()
               .required()
               .description(DESCRIPTION.providerUid),
+            email: isA.string().required().description(DESCRIPTION.email),
```

Comment:
I'm returning the email so that the email stored in local storage is the email used for third party auth (vs the email first entered in our email first form), as it might be different, especially if the user is using a private relay. Perhaps there's another way to do this?